velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -835,6 +835,26 @@ TEST_F(DateTimeFunctionsTest, plusMinusDateIntervalDayTime) {
   EXPECT_THROW(minus(baseDate, partDay), VeloxUserError);
 }
 
+TEST_F(DateTimeFunctionsTest, minusTimestampIntervalDayTime) {
+  const auto minus = [&](std::optional<Timestamp> t1,
+                         std::optional<Timestamp> t2) {
+    return evaluateOnce<int64_t>(
+        "c0 - c1",
+        makeRowVector({
+            makeNullableFlatVector<Timestamp>({t1}),
+            makeNullableFlatVector<Timestamp>({t2}),
+        }));
+  };
+  EXPECT_EQ(std::nullopt, minus(std::nullopt, std::nullopt));
+  EXPECT_EQ(1000, minus(Timestamp(1, 0), Timestamp(0, 0)));
```

Comment:
Since you only specifying seconds for timestamps, consider, simplifying this code to

```
EXPECT_EQ(1000, minus(1, 0));
EXPECT_EQ(-1000, minus(1, 2));
```